reliability-engineering/pipelines/tasks/concourse-land-workers.yml

```diff
@@ -36,7 +36,7 @@ run:
     function get_old_worker_count() {
       wc -l <workers/names
     }
-    while [ "$(get_current_worker_count)" = "$(get_old_worker_count)" ]; do
+    while [ "$(get_current_worker_count)" = "$((get_old_worker_count \* 2))" ]; do
```

Comment:
Wait, don't you want to turn this into `!=`?

Reply:
correct! will fix that now